getting-started/deployment_guide.md

```diff
@@ -24,6 +24,50 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 > [!NOTE] 
 > kArmor CLI provides a Developer Friendly way to interact with KubeArmor Telemetry. You can stream KubeArmor telemetry independently of kArmor CLI tool and integrate it with your chosen SIEM (Security Information and Event Management) solutions. [Here's a guide](https://github.com/kubearmor/kubearmor-relay-server/blob/main/README.md#streaming-kubearmor-telemetry-to-external-siem-tools) on how to achieve this integration. This guide assumes you have kArmor CLI to access KubeArmor Telemetry but you can view it on your SIEM tool once integrated.
 
+## KubeArmor EKS-plugin 
+
+> Amazon Elastic Kubernetes Service (Amazon EKS) is a managed Kubernetes service provided by Amazon Web Services (AWS) which simplifies the process of running Kubernetes clusters by handling the underlying infrastructure and operational aspects. KubeArmor’s integration with EKS can provide a user with a tool that can enforce security policies for them at the pod level as well as at the node level. Users can restrict all events related to network, process, and file using these policies.
```

Comment:
I don't think we need to explain EKS and KubeArmor here. It's a KubeArmor deployment guide so people already know about Kubernetes and Kubearmor here. Linking to the description of EKS in a hyperlink would be enough. WDYT?

Reply:
Removed unnecessary information and hyperlinked EKS-plugin official documentation.